subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -87,6 +87,13 @@ config EC_HOST_CMD_DEDICATED_THREAD
 	  The ec_host_cmd_init function creates a new thread dedicated for Host Command.
 	  Otherwise the ec_host_cmd_task function has to be called within another thread.
 
+config EC_HOST_CMD_IN_PROGRESS_STATUS
+	bool "Save status of last IN_PROGRESS command"
```

Comment:
Should this include `default y if EC_HOST_CMD_BACKEND_SHI`?

Reply:
Makes sense. Done.